### ISSUE TITLE
jar jjwt-gson implementation does not contain ServiceLoader definitions

### ISSUE THREAD

**Issue by patton73** [jwtk/jjwt#563]
The published jar in maven Central for version 0.11.0 does not contain META-INF/services package.
So the service loader does not finde the implementation.
Should be fixed and released again.

**Comment by lhazlewood** [comment]
We can never change a release that has been published to the world.  It always has to be done as a new release.  Adding to 0.11.1.

cc @bdemers

**Comment by lhazlewood** [comment]
@bdemers this is because the ServiceLoader files live under that module's `src/resources` directory, and it should have been `src/main/resources` 😛 

https://github.com/jwtk/jjwt/tree/0.11.0/extensions/gson/src

**Comment by bdemers** [comment]
Ugg 😞 
Probably a good time to add a quick test to make sure the ServiceLoader returns the expected classes.

If/when we switch to Java 1.8, we can use Google Auto Service (and not deal with creating the META-INF files by hand), but... depending on how you configure annotation processing, it could still get missed

TL;DR: a simple test is needed

**Comment by lhazlewood** [comment]
@bdemers any status on this?  Can the PR be merged?

**Comment by bdemers** [comment]
Yup, take another quick look at the PR
